app/common/update.py

```diff
@@ -253,3 +258,65 @@ def merge_local_db():
 
         logger.info(str(records_inserted) + " records were inserted into local db.")
         logger.info(str(records_updated) + " records in local db were updated.")
+        
+
+def download_dat_files():
+    config = load_user_config()
+    valid_directory = False
+    
+    if not config["config"]["installdirectory"]:
+    
+        #First, let's check the default location and see if it's installed there
+        path = 'C:/Program Files (x86)/SquareEnix/DRAGON QUEST X/Game/Content/Data'
+        
+        if os.path.exists(path):
+            update_user_config('config', 'installdirectory', path)
+        else:
+            #Doesn't exist. Let's prompt the user for it.
+            warning_message(
+            title="[dqxclarity] Couldn't Find Directory",
+            message="Could not find DQX directory. Please select\n\nthe Data folder located in your\n\nDQX install location."
+        )
+            
+            path = askdirectory()
+            
+            #Path looks legit.
+            if 'Game/Content/Data' in path:

```

Comment:
I think instead of checking if this is in the path, we should be checking for the existence of an actual file in the path that we expect to be in here. This could be prone to user error if they _somehow_ specified a path that has a similar directory structure. Very unlikely.. but hey. We want to make sure that `data00000000.win32.dat0` is in the directory for instance. We could also check for both conditionals too. Never hurts to be safe when we are about to make operations in a directory that isn't ours.